### DISCUSSION TITLE
cli: how do I subscribe to multiple topics?

### DISCUSSION THREAD

**Discussion by YAMLcase** [emqx/MQTTX#1061] [General]
I'd like to sub to multiple topics, how can I do this?  `-t topic1 topic2` and `-t topic1 -t topic2` both only show messages from the last topic

**Comment by Red-Asuka** [comment]
@YAMLcase Hi, thank you for your feedback. Unfortunately MQTT X CLI does not support multi-topic subscriptions now, but we will consider implementing it soon.

**Comment by ysfscream** [comment]
Thank you for your suggestion, we recorded it into this issue for follow-up: https://github.com/emqx/MQTTX/issues/1062

**Comment by ysfscream** [comment]
Thanks! Now available in the MQTT X CLI in 1.8.3: https://github.com/emqx/MQTTX/releases/tag/v1.8.3
